spec/features/articles_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe "Articles Youtube Section" do
+
+  describe "Youtube Video Section on Articles Show View", :vcr do
+
+    it "can show a heading and description of related videos." do
+      user = create(:user)
+      article_1 = create(:article, user_id: user.id)
+      article_2 = create(:article, user_id: user.id)
+      last_article = create(:article, user_id: user.id)
+      tags = last_article.cached_tag_list
+
+      visit "/#{user.username}"
+
+      expect(page).to have_content(article_1.title)
+      expect(page).to have_content(article_2.title)
+      expect(page).to have_content(last_article.title)
+      
+      click_on(last_article.title)
+
+      expect(page).to have_content('Related Videos')
+      expect(page).to have_content("The videos below are based on the following tags: #{tags}.")
```

Comment:
@eamouzou I like that we are confirming the video section is present, but is there any way to confirm the actual videos are present? I know Ian said it wasn't possible in Rspec, so if we can't get there by Monday, no big deal.

Reply:
@Will-Meighan, we can't really test for a fetch call with rspec. The video data is tested for in the api request test so I figure we've tested as fully as we can.

Reply:
@eamouzou Okay coo, just wanted to make sure. This is good enough for me!